base_images/Dockerfile.bamboo

```diff
@@ -17,7 +17,7 @@ yum update -y
 # install Python 3, pip, CLI utilities, session manager and other tools
 RUN \
 yum install -y jq gcc git make openssl wget zip unzip tar python3-devel nodejs yarn python3-pip awscli session-manager-plugin.rpm procps parallel && \
-pip3 install -q --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org pydoc-markdown bandit safety flake8 black isort
+pip3 install -q --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org pydoc-markdown bandit safety flake8 black isort aws-lambda-powertools
```

Comment:
It surprises me that this is needed in the Dockerfiles. Feels like adding it to the requirements.txt would be sufficient.

Reply:
hmm looks like it is not needed here since requirements file already has it